python/tvm/contrib/sdaccel.py

```diff
@@ -6,29 +6,23 @@
 
 
 @register_func("tvm_callback_sdaccel_compile")
-def compile_vhls(code, kernel):
+def compile_vhls(codes, kernels):
     """Compile Vivado HLS code for SDAccel.
 
     Parameters
     ----------
-    code : str
-        The Vivado HLS code.
+    codes : str
+        Array of the Vivado HLS codes.
 
-    kernel : str
-        The kernel to compile or link.
+    kernels : str
```

Comment:
To me "funcnames" used in codegen_vhls.cc describes this variable better.
The explanation of this variable can also be more specific, such as "Array of the kernel function names..."